moveit_core/robot_state/test/robot_state_benchmark.cpp

```diff
@@ -66,312 +61,279 @@ Eigen::Isometry3d createTestIsometry()
 }  // namespace
 
 // Benchmark time to multiply an Eigen::Affine3d with an Eigen::Matrix4d.
-static void multiplyAffineTimesMatrix(benchmark::State& st)
+static void multiplyAffineTimesMatrixNoAlias(benchmark::State& st)
```

Comment:
The NoAlias versions just use Eigen's `.noalias()` modifier, allowing to write the result of matrix multiplication directly into the result matrix instead of using an intermediate temporary (which is the default).